File: scratch/get_aov_ata.js
const https = require('https');
https.get('https://liquipedia.net/arenaofvalor/Ata', (res) => {
  let body = '';
  res.on('data', chunk => body += chunk);
  res.on('end', () => {
    let imgRegex = /<img[^>]+src=\"([^\"]+)\"[^>]*alt=\"Ata\"/ig;
    let match;
    while((match = imgRegex.exec(body)) !== null) {
      console.log('Ata URL:', match[1]);
    }
  });
});

File: scratch/get_aov_ata.py
import urllib.request
import re

req = urllib.request.Request(
    'https://liquipedia.net/arenaofvalor/Ata',
    headers={'User-Agent': 'Mozilla/5.0'}
)
try:
    with urllib.request.urlopen(req) as response:
        html = response.read().decode('utf-8')
        matches = re.findall(r'src="([^"]*ata[^"]*png)"', html, re.IGNORECASE)
        print('Matches:', matches)
except Exception as e:
    print('Error:', e)
